src/server/game/Entities/Pet/Pet.h

```diff
@@ -25,6 +25,17 @@
 #define PET_FOCUS_REGEN_INTERVAL 4 * IN_MILLISECONDS
 #define HAPPINESS_LEVEL_SIZE        333000
```

Comment:
Does happiness always exist in legion?

Reply:
Your comment wins my "Funniest comment taken out of context" award, describing the game... ;-)

Reply:
Well tkrokli.. if he wants to rewrite the pet system, he should remove unused things for this system.. to clean the core from unneccesary things

Reply:
Well can't find all corpses which went inside this handling.
Deleted a lot already.
Also i don't want to break something

Reply:
this is dead forgotten code from old cata core.